# joint of arm
float64 joint1
float64 joint2
float64 joint3
float64 joint4
float64 joint5
float64 joint6
float64 finger_joint1
float64 finger_joint2
---
# the costing time
duration time_elapsed